exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceTranslator.java

```diff
@@ -214,15 +217,19 @@ static Status toStatusProto(StatusData status) {
         statusBuilder.setCode(0);
         break;
       case UNSET:
-        statusBuilder.setCode(1);
-        break;
+        // We do not specify a code in the UNSET case.
+        return null;
       case ERROR:
+        statusBuilder.setCode(2);
```

Comment:
Can we use the `google.rpc.Code` [generated constant](http://googleapis.github.io/googleapis/java/all/latest/apidocs/com/google/rpc/Code.html#UNKNOWN) here and also on L217 and L231?